<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>saveRoi</class>
 <widget class="QWidget" name="saveRoi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save Region Of Interest (ROI)</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: rgb(42, 42, 42);
}</string>
  </property>
  <widget class="QLabel" name="saveRoiLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>-20</y>
     <width>201</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-size: 29px;
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
}</string>
   </property>
   <property name="text">
    <string>Save ROI:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="roiFolderPathLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: rgba(255, 255, 255, 0);
	color: white;
	font-size: 17px;
}</string>
   </property>
   <property name="text">
    <string>Folder to save ROI in:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="newFolderPathInput">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	background-color: rgb(249, 249, 249);
	color: black;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="chooseFolderButton">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>170</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: white;
	font-size: 16px;
	background: rgb(90, 37, 255);
	border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Choose Folder</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearFolderButton">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>170</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: white;
	font-size: 16px;
	background: rgb(90, 37, 255);
	border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Clear Folder</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveRoiButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>440</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: white;
	font-size: 16px;
	background: rgb(90, 37, 255);
	border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="dataSavedSuccessfullyLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>421</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: rgb(23, 255, 0);
	font-size: 30px;
	background-color: rgba(255, 255, 255, 0);
}</string>
   </property>
   <property name="text">
    <string>ROI saved successfully!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="newFileNameLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: rgba(255, 255, 255, 0);
	color: white;
	font-size: 17px;
}</string>
   </property>
   <property name="text">
    <string>Type name of new file:
(no spaces,  must end in &quot;.csv&quot;)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="newFileNameInput">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	background-color: rgb(249, 249, 249);
	color: black;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="fileNameErrorLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>320</y>
     <width>421</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: rgb(255, 0, 23);
	font-size: 20px;
	background-color: rgba(255, 255, 255, 0);
}</string>
   </property>
   <property name="text">
    <string>ERROR: Filename must not contain a space
and must end in &quot;.csv&quot;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="fileNameWarningLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>421</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: rgb(255, 0, 23);
	font-size: 20px;
	background-color: rgba(255, 255, 255, 0);
}</string>
   </property>
   <property name="text">
    <string>WARNING: If file of specified folder and filename
already exists,  that file will be
deleted and overwritten</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
